### ISSUE TITLE
Improvements to Inspector

### ISSUE THREAD

**Issue by Tyrannicus100BC** [Unity-Technologies/AssetBundles-Browser#3]
# Background

One of the main use cases for a tool like AssetBundles-Browser on my project is to help designers understand what is included in the AssetBundles they create. We put large portions of our game's UI into asset bundles as prefabs. This allows us to have a very customizable game through DLC without having to go through app store submissions.

Generally, UGUI is easy enough to use that we have designers (non-engineers) build screens. What is difficult for them, is to understand what assets get included in the final bundle through automatic dependency tracking and through sprite sheeting. Often, a designer will use a small image in their UI, not realizing that the image is on a large shared sprite sheet. This results in what should be a small asset bundle being surprisingly small.

This is an honest mistake on their part and one they should fix. However, without AssetBundles-Browser, the process for the designer to track down their mistake is very difficult. They either go through their prefab component by component checking sprite tag settings, or parse the cryptic asset bundle build output in the editor log.

AssetBundles-Browser is the perfect tool to allow my designers to understand the content of the bundles they author.

# Problem

Right now, the most useful section of the bundle inspector for my purpose is the Preload Table. However, it does a bad job of organizing the content. In the following screenshot, there is a single prefab inside the bundle. The prefab has two image components that reference images on two different sprite sheet. However, the UI doesn't make this very obvious:

![spritesheetsuggestion](https://cloud.githubusercontent.com/assets/4694537/23045435/71bf447c-f459-11e6-88e1-085885e46f0b.png)

# Suggestions
*First:* Differentiate included Objects vs object contents. For example, the Prefab is a unity Object that is included. So are the two sprite sheets. The Unity object has contents (the various components that makeup that object hierarchy). The spritesheets each have contents, the images included in them.

*Second:* Is there anything you can do to estimate the size of asset in the bundle? For example, one of the spritesheets is 64x64 (tiny) and another is 2048x2048 (much larger). It would be great to see KB / MB here.

**Comment by alffanclub** [comment]
Hi @Tyrannicus100BC, just wanted to let you know that we haven't forgotten you. We pulled the inspector feature because it was quite buggy and we wanted to polish up the rest of the tool before revisiting it. I'm hoping to bring this back soon, and when I do, will definitely update this ticket, and try to use it's feedback as input.

**Comment by alffanclub** [comment]
Hey @Tyrannicus100BC, I've spent a bunch of time trying to work out some way to do what you're asking, but it just isn't possible to do in a reasonable way.  Unity does not expose any of that preload data.  The only ways it could be done is if either the Unity engine exposed more data, or we wrote an external tool that loaded and parsed the data in the bundle manually. The second option isn't out of the question. It just isn't something that would fit in the scope of this browser, or the scope of time I currently have.  Sorry I couldn't be more helpful!
-Bill
